File: app/src/main/java/com/intelegain/agora/model/OccassionsData.java
package com.intelegain.agora.model;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

/**
 * Created by suraj.m on 6/9/17.
 */

public class OccassionsData {

    @SerializedName("OccassionFor")
    @Expose
    private String occassionFor;
    @SerializedName("OccassionDate")
    @Expose
    private String occassionDate;
    @SerializedName("OccassionName")
    @Expose
    private String occassionName;
    @SerializedName("OccasionImageUrl")
    @Expose
    private String occasionImageUrl;

    public String getOccassionFor() {
        return occassionFor;
    }

    public void setOccassionFor(String occassionFor) {
        this.occassionFor = occassionFor;
    }

    public String getOccassionDate() {
        return occassionDate;
    }

    public void setOccassionDate(String occassionDate) {
        this.occassionDate = occassionDate;
    }

    public String getOccassionName() {
        return occassionName;
    }

    public void setOccassionName(String occassionName) {
        this.occassionName = occassionName;
    }

    public String getOccasionImageUrl() {
        return occasionImageUrl;
    }

    public void setOccasionImageUrl(String occasionImageUrl) {
        this.occasionImageUrl = occasionImageUrl;
    }
}

File: app/src/main/java/com/intelegain/agora/model/OccassionsData.kt
package com.intelegain.agora.model

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

/**
 * Created by suraj.m on 6/9/17.
 */
class OccassionsData {
    @SerializedName("OccassionFor")
    @Expose
    var occassionFor: String? = null
    @SerializedName("OccassionDate")
    @Expose
    var occassionDate: String? = null
    @SerializedName("OccassionName")
    @Expose
    var occassionName: String? = null
    @SerializedName("OccasionImageUrl")
    @Expose
    var occasionImageUrl: String? = null

}